src/components/Dashboard/DashboardSettings/index.jsx

```diff
@@ -80,6 +80,14 @@ function DashboardSettings() {
             >
               <Input placeholder="Viewlift Watches Fetch Limit" type="number" />
             </Form.Item>
+            <Form.Item
+              className="dash-settings-input"
+              label="Stripe API Key"
+              name="stripeKey"
+              rules={validationMessages.stripeApiKey}
+            >
+              <Input placeholder="Stripe API Key" type="text" />
```

Comment:
Make the input type = `password`
```suggestion
              <Input type="password" placeholder="Stripe API Key" type="text" />
```